### PR TITLE
update flightgear to 2020.3.5

### PR DESCRIPTION
**Pull request by violetmage**
new stable

new boost stable

duplicate of #27 
(accidentally closed)

### CODE REVIEW

**Comment by violetmage** [comment]
build have fail, with new boost

```
$ cat org_flightgear_Flightgear_build.log | grep -i deprecated | wc -l
13392
```

am uncertain, if is to become problem...

File: org.flightgear.FlightGear.appdata.xml [hunk 1]
```diff
@@ -5,68 +5,69 @@ BugReportURL: https://sourceforge.net/p/fgrun/support-requests/9/
...
-	<url type="help">https://flightgear.sourceforge.net/getstart-en/getstart-en.html</url>
-	<url type="translate">https://wiki.flightgear.org/Howto:Translate_FlightGear</url>
-	<releases>
-		<!-- When updating flightgear, add a new line below (don't remove the older ones) and sort them newest-first -->
-		<release date="2020-12-01" version="2020.3.4" />
-		<release date="2020-11-23" version="2020.3.3" />
-		<release date="2020-05-26" version="2020.1.3" />
-		<release date="2019-01-29" version="2018.3.2" />
-	</releases>
+    <id>org.flightgear.FlightGear</id>
```
Comment (Vanadiae) [inline org.flightgear.FlightGear.appdata.xml:8]: Please avoid reindenting in a different style, it makes the diff horrible.

**Review by Vanadiae — CHANGES_REQUESTED** [review]

**Comment by Vanadiae** [comment]
> build have fail, with new boost
> 
> ```
> $ cat org_flightgear_Flightgear_build.log | grep -i deprecated | wc -l
> 13392
> ```
> 
> am uncertain, if is to become problem...

well, we're just packaging flightgear as a flatpak, anything related to its code is not something we can fix here.

File: org.flightgear.FlightGear.appdata.xml (hunk 1 above)
Comment (violetmage) [reply org.flightgear.FlightGear.appdata.xml]: gnome-builder have strange way of configure tabsize...

however, making excuse is no help

is better now

Comment (Vanadiae) [reply org.flightgear.FlightGear.appdata.xml]: IIRC, Builder uses the same tab style/size as the rest of the document when you press Enter at the end of the line. Anyway it's fixed so all good :)

**Comment by Vanadiae** [comment]
Overall it looks good, I'll just test it today or during the end of the week, and merge it if it's works alright.

**Comment by Vanadiae** [comment]
LGTM from my testing. Thanks again for your contributions!

**Pull request merged**